test/sealer_apply_test.go

```diff
@@ -84,148 +84,153 @@ var _ = Describe("sealer apply", func() {
-		rawCluster.Spec.Image = settings.TestNydusImageName
-
-		Context("check regular scenario that provider is bare metal, executes machine is master0", func() {
-			var tempFile string
-			BeforeEach(func() {
-				tempFile = testhelper.CreateTempFile()
-			})
-
-			AfterEach(func() {
-				testhelper.RemoveTempFile(tempFile)
-			})
-			It("init, scale up, scale down, clean up", func() {
-				By("start to prepare infra")
-				cluster := rawCluster.DeepCopy()
-				cluster.Spec.Provider = settings.AliCloud
-				cluster.Spec.Image = settings.TestNydusImageName
-				cluster = apply.CreateAliCloudInfraAndSave(cluster, tempFile)
-				defer apply.CleanUpAliCloudInfra(cluster)
-				sshClient := testhelper.NewSSHClientByCluster(cluster)
-				testhelper.CheckFuncBeTrue(func() bool {
-					err := sshClient.SSH.Copy(sshClient.RemoteHostIP, settings.DefaultSealerBin, settings.DefaultSealerBin)
-					return err == nil
-				}, settings.MaxWaiteTime)
-
-				By("start to init cluster")
-				apply.GenerateClusterfile(tempFile)
-				apply.SendAndApplyCluster(sshClient, tempFile)
 				apply.CheckNodeNumWithSSH(sshClient, 2)
-
-				By("Wait for the cluster to be ready", func() {
-					apply.WaitAllNodeRunningBySSH(sshClient.SSH, sshClient.RemoteHostIP)
-				})
-
-				By("Use join command to add 3master and 3node for scale up cluster in baremetal mode", func() {
-					cluster.Spec.Nodes.Count = "3"
-					cluster.Spec.Masters.Count = "3"
-					cluster = apply.CreateAliCloudInfraAndSave(cluster, tempFile)
-					//waiting for service to start
-					time.Sleep(10 * time.Second)
-					joinMastersIPStrs := utilsnet.IPsToIPStrs(cluster.Spec.Masters.IPList[1:])
-					joinMasters := strings.Join(joinMastersIPStrs, ",")
-					joinNodesIPStrs := utilsnet.IPsToIPStrs(cluster.Spec.Nodes.IPList[1:])
-					joinNodes := strings.Join(joinNodesIPStrs, ",")
-					//sealer join master and node
-					apply.SendAndJoinCluster(sshClient, tempFile, joinMasters, joinNodes)
-					//add 3 masters and 3 nodes
-					apply.CheckNodeNumWithSSH(sshClient, 6)
-				})
-
-				By("start to scale down cluster")
 				cluster.Spec.Nodes.Count = "1"
 				cluster.Spec.Nodes.IPList = cluster.Spec.Nodes.IPList[:1]
-				cluster.Spec.Masters.Count = "3"
+				cluster.Spec.Masters.Count = "1"
+				cluster.Spec.Provider = settings.CONTAINER
 				apply.WriteClusterFileToDisk(cluster, tempFile)
-				apply.SendAndApplyCluster(sshClient, tempFile)
-				apply.CheckNodeNumWithSSH(sshClient, 4)
+				// sealer apply scale down is not supported, if supported, we can use this
+				// apply.SendAndApplyCluster(sshClient, tempFile)
+
 				By("start to delete cluster")
-				err := sshClient.SSH.CmdAsync(sshClient.RemoteHostIP, apply.SealerDeleteCmd(tempFile))
+				err = sshClient.SSH.CmdAsync(sshClient.RemoteHostIP, apply.SealerDeleteAll())
 				testhelper.CheckErr(err)
 			})
 
 		})
+
+		// Context("check regular scenario that provider is bare metal, executes machine is not master0", func() {
```

Comment:
if those code is no longer used, could we just delete it ?

Reply:
Yes, I commented out these code because they will cost more time for local e2e test.